project_euler/problem_11/sol2.py

```diff
@@ -38,7 +39,8 @@ def solution():
     >>> solution()
     70600674
     """
-    with open("grid.txt", "r") as f:
+    path = os.path.split(os.path.realpath(__file__))
```

Comment:
More intuitive to do __is.path.dirname()__ rather than .split()[0].

Reply:
I changed "os.path.split(os.path.realpath(__file__))" to "os.path.dirname(__file__)". Thanks for suggestion.